spec/devengo/api/transactions_service_spec.rb

```diff
@@ -39,39 +51,96 @@
     end
   end
 
-  describe "find transaction" do
-    include_context "with stub request",
-                    path: "accounts/acc_fYpgX5Ytdxzexuf61lFmw/transactions/txn_3NuWZw8zSY2LNMoNXgVwaR",
-                    resource: "transactions/find"
-
-    let!(:transaction) do # rubocop:disable RSpec/LetSetup
-      transactions_service.find(account_id: "acc_fYpgX5Ytdxzexuf61lFmw", transaction_id: "txn_3NuWZw8zSY2LNMoNXgVwaR")
-    end
-
-    it_behaves_like "builds correct request",
-                    method: :get,
-                    path: "accounts/acc_fYpgX5Ytdxzexuf61lFmw/transactions/txn_3NuWZw8zSY2LNMoNXgVwaR"
-
-    it_behaves_like "transaction expects"
-  end
-
   describe "list transactions" do
     include_context "with stub request",
                     path: "accounts/acc_fYpgX5Ytdxzexuf61lFmw/transactions",
                     resource: "transactions/list"
 
     let!(:transactions) { transactions_service.list(account_id: "acc_fYpgX5Ytdxzexuf61lFmw") }
-    let(:transaction) { transactions.first }
 
     it_behaves_like "builds correct request",
                     method: :get,
                     path: "accounts/acc_fYpgX5Ytdxzexuf61lFmw/transactions"
 
-    it_behaves_like "transaction expects"
+    it "transactions with expected data" do
+      transactions_array = []
```

Comment:
This is weird, but we haven't been able to find another way to access all elements (not only the first) in transactions. Any idea is more then welcome.

Reply:
@raulvillares Umm... I don't know if I understand what the problem is here. If I'm reading this correctly, we are dealing with a `BaseCollection` that includes `Enumerable` and implements `each`, so its items should be available via `#entries`, `#to_a` or even the more indirect `#.each.to_a` as `each` always returns an `Enumerator`. 

What am I missing?

Reply:
You aren't missing a thing except my ignorance on basic ruby structures 😅 
Just changed it using #entries